api/proto/info/info.proto

```diff
@@ -286,6 +286,7 @@ message ZInfoDevice {
   uint32 ncpu = 7;
   uint64 memory = 8;   // in Mbytes
   uint64 storage = 9;  // in MBytes for the currently active image filesystem
+  int64 powerCycleCounter = 10;
```

Comment:
How about adding a comment that it is set to -1 if the hardware doesn't support such a counter?

Reply:
Added comment.